src/Products/PluginIndexes/BooleanIndex/BooleanIndex.py

```diff
@@ -137,8 +137,8 @@ def removeForwardIndexEntry(self, entry, documentId, check=True):
                 raise
             except Exception:
                 LOG.exception(
-                    '%s: unindex_object could not remove documentId %s '
-                    'from index %s. This should not happen.' % (
+                    '{0}: unindex_object could not remove documentId {1} '
+                    'from index {2}. This should not happen.'.format(
```

Comment:
Although on other places it is beneficial to change to the `.format()` version, for logging it might be better to add the parameter as `*args` to the `LOG.exception()` as it gets formatted later on anyway. But as the initial case did not used this way, I can live with that here.